### PR TITLE
FIX: "Easy Netplay" final touches: Removed slow chksum from GLO - QOL fixes - File size check

### PR DESCRIPTION
**Pull request by Aemiii91**
## Problem

GLO is currently calculating the chksum of both the current rom and core - this is very slow for PSX.

## Changes

* Cksum generation removed from GLO (But GLO still makes half of the cookie, (rom path/core path))
* Cksum generation now added to the host side script but delays mitigated by checking file size for larger files, less accurate but much faster.
* Added filesize check to host script (If any of the files are above 25mb we write the file size instead of the cksum) - Cookie label will become [$typesize] instead of [$typechksum]
* Added filesize check to join script (If any of the files are above 25mb we match the file size instead of the cksum)
* Add proxy scripts to Favourites/Recents/RApp (moved parent scripts to .tmp_update/script/easynetplay)
* Aligned log output with host or client
* Fixed intermittent issue with wifi re-association.
* Replaced "exit" with "cleanup" to attempt to restore the wifi state if we don't actually do a full cycle (we fail somewhere critical) - this will also clear bad network ids from wpa_supp so we don't bloat the file with stale networks.
* Improved speeds for both hosting and joining
* Fixed logging output and filename
* GLO now hides scripts not available to the MM (such as networking)
* Fixed host side hanging on exit of retroarch
* Script names updated
* Clientside now confirms joining

## To-do:

- [x] Allow the process to be killed at any point
- [ ] ~~Add custom RetroArch config overrides for Easy Netplay~~ Tested on stock config.
- [x] Add migration script to remove old romscripts (see #991)

## Tested with:

PSX Rearmed -> Digimon world 2 - to validate delay on larger roms
TGBDual GB - Balloon kid - to validate we haven't regressed on smaller roms/cores
mGBA GBA - Pokemon Leaf (Spectate) to validate we haven't regressed on smaller roms/cores
Bubble bobble - NES - FCEUMM - Netplay works flawlessly for 2p on this game

We could really do with a progress bar on the rom download/core download screen if PSX ever becomes viable for netplay as the roms are huge.

### CODE REVIEW

File: static/build/.tmp_update/script/easynetplay/netplay_server.sh [hunk 1]
```diff
@@ -193,11 +193,14 @@ build_infoPanel() {
     local title="$1"
     local message="$2"
     
-    infoPanel --title "$title" --message "$message" --auto
+    infoPanel --title "$title" --message "$message" --persistent &
+    sleep 0.5
```
Comment (Aemiii91) [inline static/build/.tmp_update/script/easynetplay/netplay_server.sh:197]: You should move the `sleep` after `sync` - to ensure infoPanel has closed before continuing

Comment (XK9274) [reply static/build/.tmp_update/script/easynetplay/netplay_server.sh]: Resolved in https://github.com/OnionUI/Onion/pull/969/commits/9562a6583dbcf500e0a78bd976ebe49fc00214b7

**Review by XK9274 — APPROVED** [review]
Tested relevant scripts 👍 A lot faster now!

**Comment by XK9274** [comment]
Done quite a lot of testing on this today. Did a fresh build on one MMP and used the retroarch.cfg from this fresh build on all 3. 

## Tested on multiple games:
F1 Race - TGB Dual - 2 player - OK. 
Balloon kid - TGB Dual - 2 player - OK
Xmen - FB Neo - 3 player - OK 

No patching was needed, although the user must select a netplay capable core as their default currently.

**Review by XK9274 — APPROVED** [review]
Approving pending a full final test before release

**Pull request merged**